airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/scheduling/GeographyMapper.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.temporal.scheduling;
+
+import io.airbyte.config.Geography;
+
+/**
+ * Maps a {@link Geography} to a Temporal Task Queue that should be used to run syncs for the given
+ * Geography.
+ */
+public interface GeographyMapper {
```

Comment:
Thinking out loud:

Would a better name not include Geography? Thinking about the future where we want the input to `getTaskQueue` to not be tied to geography. Maybe QueueMapper?

Landing what we have now and evolving this later is also okay to me.